js/search/search.js

```diff
@@ -69,8 +69,8 @@ class Search {
     init() {
         libSymphonySearch.symSEInit();
         libSymphonySearch.symSEEnsureFolderExists(searchConfig.FOLDERS_CONSTANTS.INDEX_PATH);
-        Search.deleteIndexFolders(searchConfig.FOLDERS_CONSTANTS.TEMP_REAL_TIME_INDEX);
-        Search.deleteIndexFolders(searchConfig.FOLDERS_CONSTANTS.TEMP_BATCH_INDEX_FOLDER);
+        Search.deleteIndexFolders('REAL_TIME_INDEX');
```

Comment:
@VikasShashidhar We will no longer be having the data folder when we implement SEARCH-729. This PR is just to prevent deleting other files which are not related to search.